kalite/distributed/views.py

```diff
@@ -249,3 +250,15 @@ def handler_500(request):
         "value": unicode(value),
     }
     return HttpResponseServerError(render_to_string("distributed/500.html", context, context_instance=RequestContext(request)))
+
+def sphinx_docs(request, doc_url):
```

Comment:
Have you tried just serving the html files statically? Or do we really need to have a separate view?